### PR TITLE
RAILS PROJECT: FORMS

### PR DESCRIPTION
**Pull request by krys2fa**
## Reformer

In this project:
- we created a user model and validations.
- we built a basic HTML form.
- we built a basic HTML form using the rails form tag.
- we built a basic HTML form using the rails form_for tag.
- we displayed on our views.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<%= form_for @user do |f| %>
+
+<ul>
+  <% @user.errors.full_messages.each do |error| %>
+    <li><%= error %></li>
+  <% end %>
+  </ul>
+
+
+  Username:
```
Comment (Ispirett) [inline app/views/users/new.html.erb]: Instead of using `Username` you can use `  <%= f.label :username%>`

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <%= f.text_field :username, placeholder:"Your name here."%>
+  Email:
```
Comment (Ispirett) [inline app/views/users/new.html.erb]: Follow the above example.

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <%= f.email_field :email, placeholder:"example@example.com" %>
+  Password:
```
Comment (Ispirett) [inline app/views/users/new.html.erb]: Follow the above example.

**Review by Ispirett — CHANGES_REQUESTED** [review]
## Status changes required: 

Great job @krys2fa @SafaErden  you guys did great :+1: .

Please check my comments below and make the necessary changes.

**Review by mosesogwo — APPROVED** [review]
**Hi @krys2fa @SafaErden:clap: Congratulations!**

You have done well with the requested changes. Your project is approved

**Good luck in your next project:stars:**

### Status: Approved :heavy_check_mark:

**Pull request merged**